File: benchmarks/list_sum.rb
t0 = Process.clock_gettime(Process::CLOCK_MONOTONIC, :nanosecond)
acc = []
i = 0
while i < 3000000
    acc << i
    i += 1
end
total = 0
acc.each do |x|
    total += x
end
puts total
puts "bench_ns=#{Process.clock_gettime(Process::CLOCK_MONOTONIC, :nanosecond) - t0}"

File: benchmarks/list_sum.js
const { performance } = require('perf_hooks');

const t0 = performance.now();
const acc = [];
for (let i = 0; i < 3000000; i++) {
    acc.push(i);
}
let total = 0;
for (let i = 0; i < acc.length; i++) {
    total += acc[i];
}
console.log(total);
console.log(`bench_ns=${Math.round((performance.now() - t0) * 1e6)}`);
